### PR TITLE
Remove function prototypes from backend pt5

### PR DESCRIPTION
**Pull request by dkorpel**

### CODE REVIEW

File: compiler/src/dmd/backend/code.d [hunk 1]
```diff
@@ -85,15 +85,10 @@ union evc
 
 /********************** PUBLIC FUNCTIONS *******************/
 
-code *code_calloc();
-void code_free(code *);
-void code_term();
+public import dmd.backend.dcode : code_calloc, code_free, code_term, code_chunk_alloc, code_list;
```
Comment (thewilsonator) [inline compiler/src/dmd/backend/code.d:88]: trailing ws

File: compiler/src/dmd/backend/code.d [hunk 2]
```diff
@@ -191,35 +184,11 @@ void nteh_monitor_epilog(ref CodeBuilder cdb,regm_t retregs);
...
-void cgreg_used(uint bi,regm_t used);
-void cgreg_spillreg_prolog(block *b,Symbol *s,ref CodeBuilder cdbstore,ref CodeBuilder cdbload);
-void cgreg_spillreg_epilog(block *b,Symbol *s,ref CodeBuilder cdbstore,ref CodeBuilder cdbload);
-int cgreg_assign(Symbol *retsym);
-void cgreg_unregister(regm_t conflict);
-
-// cgsched.c
-void cgsched_block(block *b);
+public import dmd.backend.cgen;
+public import dmd.backend.cgreg : cgreg_init, cgreg_term, cgreg_reset, cgreg_used,
```
Comment (thewilsonator) [inline compiler/src/dmd/backend/code.d:188]: ditto

File: compiler/src/dmd/backend/code.d [hunk 3]
```diff
@@ -320,30 +289,9 @@ struct FuncParamRegs
...
-    segidx_t cseg;
-    int STACKALIGN;
-    int TARGET_STACKALIGN;
-    LocalSection Para;
-    LocalSection Fast;
-    LocalSection Auto;
-    LocalSection EEStack;
-    LocalSection Alloca;
-}
+public import dmd.backend.cg : BPRM, FLOATREGS, FLOATREGS2, DOUBLEREGS,
```
Comment (thewilsonator) [inline compiler/src/dmd/backend/code.d:292]: ditto

File: compiler/src/dmd/backend/code.d (hunk 1 above)
Comment (dkorpel) [reply compiler/src/dmd/backend/code.d:88]: Heh, forgot to turn the setting back on after https://github.com/libmir/mir-algorithm/pull/464#issuecomment-1532256617 :upside_down_face:

**Review by thewilsonator — APPROVED** [review]
LGTM

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**